api/v1beta1/disruption_cron_webhook_test.go

```diff
@@ -136,21 +166,75 @@ var _ = Describe("DisruptionCron Webhook", func() {
 				})
 			})
 
+			DescribeTable("should return an error when disruption cron name is invalid",
+				func(name, expectedError string) {
+					// Arrange
+					disruptionCron := makeValidDisruptionCron()
+					disruptionCron.Name = name
+
+					// Act
+					warnings, err := disruptionCron.ValidateCreate()
+
+					// Assert
+					Expect(warnings).To(BeNil())
+					Expect(err).Should(HaveOccurred())
+					Expect(err).To(MatchError(ContainSubstring(expectedError)))
+				},
+				Entry("when name is an empty string", "", "disruption cron name must be specified"),
+				Entry("when name is too long", "a-very-long-disruptioncron-name-of-48-characters", "disruption cron name exceeds maximum length: must be no more than 47 characters"),
+				Entry("when name is not a valid Kubernetes label", "invalid-name!", "disruption cron name must follow Kubernetes label format"),
+			)
+
+			When("disruption cron schedule is invalid", func() {
```

Comment:
Why not using a `DescribeTable` to verify more cases?

Reply:
We are using the [cron](https://pkg.go.dev/github.com/robfig/cron) package. I might be wrong, but as far as I understand, we don't test the correctness of external packages. I used one use case just to verify that it actually works.